revisions/N0/src/text/dialog_2.asm

```diff
@@ -142,9 +168,13 @@ Dialog0B3::
 
 Dialog0B4::
     db "HO HO HO!       "
-    db "I'm your bad guy"
-    db "this time!!     "
+    db "Jeg er slemming-"
```

Comment:
Hva med «Jeg er dagens slemming»? Hehe

Reply:
monster of the week liksom?

Reply:
kan du foreslå
172 db "en på menyen i  "
173 db "dag!! HO HO HO!@"
?
jeg kan ikke foreslå på min egen PR tror jeg